### DISCUSSION TITLE
I lost my license and relative email

### DISCUSSION THREAD

**Discussion by gorypro** [aclap-dev/video-downloadhelper#445] [License & Purchase requests]
### Your question

Hello, I purchased a license a few years ago but I've lost the data. I can't remember which email I registered it with. It should be one of these emails. Is there a way to recover it?

gia_lux@hotmail.it
gianluca.maio98@hotmail.it
gianluca.maio13@gmail.com
maiogianluca@yahoo.com

Please, help me!!!

### Addon & CoApp version and details

version: 8.2.2.9
channel: stable
build date: 2024-03-19
build options: linuxlic: false, noyt: true, browser: chrome
lang: it
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/124.0.0.0 Safari/537.36
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\maiog",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by jcfrog** [comment]
Hi,
We sent you an email.

**Reply by gorypro** [reply]
Hi, sorry, I just saw this response now. After all this, I can't find any email, maybe I deleted it. Could you resend it?

**Reply by jcfrog** [reply]
An email has been resent to [gia_lux@hotmail.it](mailto:gia_lux@hotmail.it)
